.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: java
```

Comment:
please see https://github.com/fbricon/wildfly-hive/blob/master/.travis.yml for better settings (including caching maven bits)